util/index.js

```diff
@@ -8,3 +10,33 @@ export function preprocessJsonInput(text) {
   }
 }
 export {parseSite} from './page-parser.js';
+
+export function createTestFile(filePath) {
+  const boilerPlate = `import { test, expect } from '@playwright/test';
+
+test('generated test', async ({ page }) => {
+`;
+
+  fs.writeFileSync(filePath, boilerPlate.trim() + '\n');
+}
+
+export function appendToTestFile(userInput, generatedCode, filePath) {
+  const indentation = 1;
+  let formattedCode = `\t// ${userInput}\n`;
+
+  // Split the code into lines and format each line
+  const lines = generatedCode.split('\n');
+  for (const line of lines) {
+    formattedCode += formatCodeLine(line, indentation);
+  }
+
+  fs.appendFileSync(filePath, formattedCode, 'utf8');
+}
```

Comment:
Looks like indentation is always 1. We can simplify it to this
```suggestion
  let formattedCode = `\t// ${userInput}\n`;

  // Split the code into lines and format each line
  const lines = generatedCode.split('\n');
  for (const line of lines) {
    formattedCode += `\t${line.trim()}\n`;
  }

  fs.appendFileSync(filePath, formattedCode, 'utf8');
}
```